src/BridgeIHMCORS.cpp

```diff
@@ -147,16 +172,37 @@ void BridgeIHMCORS::run()
                     m_robotFeedback.jointStates()[jnt].q() = m_jointPositionsFromYARP[jnt];
                     m_robotFeedback.jointStates()[jnt].qd() = m_jointVelocitiesFromYARP[jnt];
                 }
-                m_robotFeedback.jointStates()[jnt].tau() = m_jointTorquesFromYARP[jnt];
+               
+               m_robotFeedback.jointStates()[jnt].tau() = m_jointTorquesFromYARP[jnt];
+               // std::cerr << "m_robotFeedback message updated " << std::endl;
+            } 
 
-                // std::cerr << "m_robotFeedback message updated " << std::endl;
-            }
 
+           // serialize m_robotFeedback
+           Cdr m_robotFeedback_ser(fastBuffer);
+           m_robotFeedback_ser << m_robotFeedback;
```

Comment:
nitpick: use methods instead of operator if possible. The code will be clearer.
In this case, there is `serialize`